lang/java/avro/src/test/java/org/apache/avro/reflect/TestReflect.java

```diff
@@ -1048,4 +1050,17 @@ public void testAvroDefault() {
           +"{\"name\":\"foo\",\"type\":\"int\",\"default\":1}]}");
   }
 
+  private static class NullableBytesTest {
+    @Nullable
+    byte[] bytes = "foo".getBytes();
+  }
+
+  @Test
+  public void testNullableByteArray() throws IOException {
```

Comment:
Instead of documenting the test, I rewrote it. It asserts for the output in the updated version